### DISCUSSION TITLE
File transfer with RustDesk

### DISCUSSION THREAD

**Discussion by dragonfly45-hue** [rustdesk/rustdesk#9753] [Q&A]
Hallo,
When I start the file transfer, the following message appears: no active console user logged on, please connect and logon first
I can see the file directory of the other side, after confirming the message the connection breaks down.
What do I have to do to start a transfer?

Best Regards
Dietmar

**Comment by xlionjuan** [comment]
You should provide the OS you're using, if Linux, Wayland or X11?

**Comment by dragonfly45-hue** [comment]
Hi,
I use ArchLinux with Wayland desktop and Linux Mint with Cinamon-Desktop.

**Comment by xlionjuan** [comment]
> ArchLinux with Wayland desktop

What DE? Try X11

> Linux Mint with Cinamon-Desktop

Cinnamon is still bad on Wayland, so I assume you're using X11

**Comment by canonex** [comment]
Hi,
I'm on Debian Linux with Gnome, on both machines (Debian 12 and Debian 13 testing), I confirm the issue.
Thank you,
Riccardo

**Comment by dragonfly45-hue** [comment]
Hallo,
I have the file transfer between various Linuxsystems tried out, also between 2 Windows systems.
The file transfer generally does not work.

Can no one help me?

See previous post by canonex, it doesn't work there either.

Perhaps this is a mistake?

Best Regards
Dietmar
+

**Reply by xlionjuan** [reply]
On Linux, it has lots of factors with no doubts, but Windows to Windows also have problem......is your connection okay?

**Comment by dragonfly45-hue** [comment]
Hi,
What does the error message mean? no active console user logged on, please connect and logon first
Everything works with Teamviewer

**Reply by xlionjuan** [reply]
On windows, have you logged in?
